File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/company/dtos/CreateIntervalResponse.java
package com.slotify.backend.spring.company.dtos;


import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.time.LocalDateTime;
import java.util.UUID;

@Builder
@Getter
@ToString
public class CreateIntervalResponse {

    private UUID intervalId;
    private LocalDateTime startDatetime;
    private LocalDateTime endDatetime;
    private Integer maxConcurrentServices;

}

File: frontend/app/src/main/java/com/raj/slotify/dtos/company/CreateIntervalResponse.kt
package com.raj.slotify.dtos.company

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class CreateIntervalResponse(

    @SerializedName("intervalId")
    val intervalId: UUID,

    @SerializedName("startDateTime")
    val startDateTime: String,

    @SerializedName("endDateTime")
    val endDateTime: String,

    @SerializedName("maxConcurrentServices")
    val maxConcurrentServices: Int

)
